### PR TITLE
Add GLADE CMIP6 example

### PR DESCRIPTION
**Pull request by andersy005**
Adapted from the original collection residing in https://github.com/NCAR/intake-esm-datastore/blob/master/catalogs/glade-cmip6.json

### CODE REVIEW

File: examples/glade-cmip6.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  ],
+  "id": "glade-cmip6",
+  "title": "CMIP6 Archive on NCAR's GLADE",
+  "description": "This is an ESM collection for CMIP6 data accessible on the NCAR's GLADE filesystem",
+  "extent": {
+    "spatial": {
+      "bbox": [[-180, -90, 180, 90]]
+    },
+    "temporal": {
+      "interval": [[null, null]]
```
Comment (rabernat) [inline examples/glade-cmip6.json]: Let's just try to fill this in. 0000-8000 ?

File: examples/glade-cmip6.json [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    {
+      "href": "https://pcmdi.llnl.gov/CMIP6/TermsOfUse/TermsOfUse6-1.html",
+      "type": "text/html",
+      "rel": "license",
+      "title": "CMIP6: Terms of Use"
+    }
+  ],
+  "assets": {
+    "thumbnail": {
+      "href": "cmip6-thumbnail.png",
```
Comment (rabernat) [inline examples/glade-cmip6.json:41]: Is this a relative link? I have used https://www.carbonbrief.org/wp-content/uploads/2019/11/cmip6_mips.jpg in the past.

File: examples/glade-cmip6.json [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+      "type": "image/png",
+      "roles": ["thumbnail"]
+    },
+    "catalog": {
+      "href": "glade-cmip6.csv.gz",
+      "title": "Catalog",
+      "description": "Path to a CSV file with the catalog contents.",
+      "type": "text/csv",
+      "roles": ["esm-catalog"],
+      "esm:column_name": "path"
```
Comment (rabernat) [inline examples/glade-cmip6.json:52]:
```suggestion
      "esm:data_column_name": "path"
```

Comment (rabernat) [reply examples/glade-cmip6.json:52]: Also, do we need `esm:format`?

**Comment by andersy005** [comment]
Closing this for now... I am going to remove the csv and open a new PR.

**Pull request closed without merging**